src/generics/requests.js

```diff
@@ -2,9 +2,24 @@ const request = require('request')
 const parser = require('xml2json')
 
 var get = function (url, token = '', internal_access_token = false) {
-	return new Promise(async (resolve, reject) => {
+	return new Promise((resolve, reject) => {
```

Comment:
@nevil-mathew  change this to async

Reply:
we are not using any await inside that function.so async is not required

Reply:
oh ok fine